packages/preview-middleware-client/package.json

```diff
@@ -28,7 +28,7 @@
     "dependencies": {},
     "devDependencies": {
         "@sap-ux/eslint-plugin-fiori-tools": "workspace:*",
-        "@sapui5/types": "1.117.1",
+        "@sapui5/types": "1.118.0",
```

Comment:
@tobiasqueck OK to keep this updates to the latest?

Reply:
No objections